docs/source/comparison_dl_soft_transposed.csv

```diff
@@ -0,0 +1,23 @@
+**Name**,`ivadomed <https://github.com/ivadomed/ivadomed>`_,`monai <https://github.com/Project-MONAI/MONAI>`_,`delira <https://github.com/justusschock/delira>`_,`MIC-DKFZ <https://github.com/MIC-DKFZ/medicaldetectiontoolkit>`_,`ANTsPyNet <https://github.com/ANTsX/ANTsPyNet>`_,`DLTK <https://github.com/DLTK/DLTK>`_,`MIScnn <https://github.com/frankkramer-lab/MIScnn>`_,`niftytorch <https://niftytorch.github.io/doc/>`_,`DeepNeuro <https://github.com/QTIM-Lab/DeepNeuro>`_
+**BIDS**,|yes|,|no|,|no|,|no|,|no|,|no|,|no|,|yes|,|no|
+**DL base library**,PyTorch,PyTorch,PyTorch and TensorFlow,PyTorch,Tensorflow/Keras,Tensorflow,Tensorflow/Keras,PyTorch,Tensorflow/Keras
+"**Task (segmentation, detection, classification)**","Classification, Segmentation, Detection","Segmentation, Classification","Classification, Generation, Segmentation",Detection,"Classification, Segmentation, Clustering, Registration","Classification, Segmentation,Registration",Segmentation,"Classification, Segmentation ",Segmentation
```

Comment:
this row is to verbose, and to echo @charleygros in https://github.com/ivadomed/ivadomed/pull/567#issuecomment-741329078, i suggest we squeeze it. My suggestion would be:
~~~
"**Task (*)**","Class, Seg, Detect","Class, Seg"
~~~
and below the table, have a legend:
(*): Class: Classification | Detect: Detection | Seg: Segmentation

Reply:
Done